modules/telco-upgrade-upgrading-the-olm-operators.adoc

```diff
@@ -0,0 +1,106 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/day_2_core_cnf_clusters/upgrading/telco-upgrade-completing-the-upgrade.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="telco-upgrade-upgrading-the-olm-operators_{context}"]
+= Upgrading the OLM Operators
+
+In telco, software needs to vetted before it is loaded onto a production cluster.
+Production clusters are also configured in a disconnected network, which means that they are not always directly connected to the internet.
+Because the clusters are in a disconnected network, the OpenShift Operators are configured for manual upgrade during installation so that new versions can be managed on a cluster by cluster basis.
```

Comment:
```suggestion
Because the clusters are in a disconnected network, the OpenShift Operators are configured for manual upgrade during installation so that new versions can be managed on a cluster-by-cluster basis.
```